### DISCUSSION TITLE
Adding Custom (Buildbear) Forked chains to Web3Modal

### DISCUSSION THREAD

**Discussion by malachy17** [WalletConnect/walletconnect-monorepo#3925] [Developer Support]
Is it possible to add a custom forked chain (via Buildbear) to the web3modal configuration?

I saw the following page: [https://wagmi.sh/core/chains#build-your-own](https://wagmi.sh/core/chains#build-your-own)  , regarding building one's own chain object that inherits the Chain type. However, when I tried to do this for a forked chain that I had made, I wasn't able to add the my custom chain to the array of chains that the web3modal configuraion takes in, as Typescript complained that the Chain object's id that I provided was not assignable to type:

`'59140 | 5 | 1 | 137 | 80001 | 11155111 | 56 | 97 | 42161 | 421613 | 10 | 420 | 43114 | 43113 | 8453 | 84531 | 59144'`

I got similar Typescript errors regarding the name and network. This leads me to believe that there's only a certain amount of approved chains that I can custom add to the web3modal, though this also doesn't seem to make sense, given that I would assume wagmi would thus already support importing those chains out of the box.

**Comment by glitch-txs** [comment]
interesting, could you set up a minimal reproducible example of the issue? a public repo would be great!
